app/[locale]/(dashboard)/sensors/[id]/edit/page.jsx

```diff
@@ -12,7 +12,10 @@ async function Home({ params }) {
 
   return (
     <Page>
-      <PageHeader title={`Seeing edit of id: ${params.id}`} />
+      <PageHeader
+        title={`Seeing edit of id: ${params.id}`}
+        backpageButton={true}
```

Comment:
You can simplify with `showBack`. You don't need to put `true` each time by default React set the props to `true`. It can look like : 
```jsx
<PageHeader
  title={`Seeing edit of id: ${params.id}`}
  showBack 
  />
```